### PR TITLE
fix Azure vpc peering connection import

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Fixes Azure vpc peering connection import.

### CODE REVIEW

**Comment by ivan-savciuc** [comment]
@muralibasani have you tried to import an Azure VPC manually with this patch?

File: internal/service/vpc/resource_vpc_peering_connection.go [hunk 1]
```diff
@@ -634,3 +635,11 @@ func validateVPCID(i interface{}, k string) (warnings []string, errors []error)
 
 	return warnings, errors
 }
+
+// ToNilOrStringPointer returns string pointer or nil
+func ToNilOrStringPointer(s string) *string {
```
Comment (ivan-savciuc) [inline internal/service/vpc/resource_vpc_peering_connection.go:640]: It looks we can reuse `OptionalStringPointer` instead of this function

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
I haven't tested it, since it request manual testing but code looks Ok.

File: internal/service/vpc/resource_vpc_peering_connection.go (hunk 1 above)
Comment (byashimov) [reply internal/service/vpc/resource_vpc_peering_connection.go]: Yeah, have tested too. Unfortunately due to some unknown issues I can't provide a working test. I'll try to do that one more time. Function has been renamed.

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Thanks for testing locally.

Regarding the function name, I meant to say that we already have it in the schemautil package: https://github.com/aiven/terraform-provider-aiven/blob/0c7d157d35054bcf8b57aae9774b3d78ace22db8/internal/schemautil/schemautil.go#L26

Can you use it?

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**